lib/net.js

```diff
@@ -1022,6 +1025,7 @@ function afterConnect(status, handle, req, readable, writable) {
 
   assert.ok(self._connecting);
   self._connecting = false;
+  self._sockname = null;
```

Comment:
Wouldn't it logically still be null here?  Not that an extra assignment hurts, just wondering.